devtron-labs/dashboard
# Changed select dropdown UI/UX

# Description

Highlight the whole background upon selection or already selected item from dropdown instead of showing tick mark.

[**List of select dropdown**](https://docs.google.com/spreadsheets/d/1FfPM6kqCWPpBBBtmvlbPVmJSEe8ZzZdQhuyYdatob9Y/edit?usp=sharing)

Fixes : https://github.com/devtron-labs/devtron/issues/1285

# Proposal

**Existing Selection** ->
![Existing_Selection.png](https://images.zenhubusercontent.com/61e15405a1d0b1e0e115f611/7762eecb-4010-4653-b0b2-2a3ad47e7f53)

**Expected UX/UI** ->
![New_UX.png](https://images.zenhubusercontent.com/61e15405a1d0b1e0e115f611/63af1d8f-0c02-4207-b547-ed558b3884fc)

## Type of change


# How Has This Been Tested?
All tests have been performed manually. The following are the steps,

1. Visit the select-dropdown from above list
2. Click on dropdown
3. Select and Hover on any item 
4. Observe

# Checklist:
